specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2219 @@
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ContactsProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Customer creates a contact resource for a spacecraft resource."
+    },
+    "AvailableContactsListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AvailableContacts"
+          },
+          "x-ms-identifiers": [],
+          "description": "A list of available contacts."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        }
+      },
+      "description": "Response for the ListAvailableContacts API service call."
+    },
+    "AvailableContacts": {
+      "type": "object",
+      "properties": {
+        "spacecraft": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/ResourceReference"
+            }
+          ],
+          "description": "The reference to the spacecraft resource."
+        },
+        "groundStationName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Name of Azure Ground Station."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "allOf": [
+            {
+              "$ref": "#/definitions/ContactInstanceProperties"
+            }
+          ],
+          "description": "Properties of Contact resource."
+        }
+      },
+      "description": "Customer retrieves list of Available Contacts for a spacecraft resource. Later, one of the available contact can be selected to create a contact."
+    },
+    "AvailableGroundStationListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AvailableGroundStation"
+          },
+          "description": "A list of ground station resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        }
+      },
+      "description": "Response for the AvailableGroundStations API service call."
+    },
+    "AvailableGroundStation": {
+      "type": "object",
+      "x-ms-client-flatten": true,
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "ID of groundStation."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Name of the ground station."
+        },
+        "location": {
```

Comment:
>"location": [](http://example.com/codeflow?start=8&length=11)

You get a linter error here:
[This is a partial consequence of not inheriting proxyresource.]
One way to fix it is to add the x-ms-mutability attribute -- if its accurate, that is.

Property location must have 'x-ms-mutability':['read', 'create'] extension defined.
Location: Microsoft.Orbital/stable/2022-11-01/orbital.json#L1913